dev/docker/hive/Dockerfile

```diff
@@ -45,8 +64,6 @@ RUN apt-get update && apt-get upgrade -y && apt-get install --fix-missing -yq \
 RUN mkdir /root/.ssh
 RUN cat /dev/zero | ssh-keygen -q -N "" > /dev/null && cat /root/.ssh/id_rsa.pub > /root/.ssh/authorized_keys
 
-COPY packages /tmp/packages
```

Comment:
you can use `ADD`, if use ADD in docker file for a zip file, docker builder will copy and unzip it, it will reduce the image size.

then, seems we do not need two stage build.

Reply:
Thanks for you advise! I used ADD in the beginning, but I found that we need --strip-component to remove outer directory, ADD cannot do this.

Reply:
Yes, `--strip-component` did not worked in `ADD`, so I use a soft link in Doris Image

Reply:
Okay, I'll try the soft-link approach, thanks 😀